packages/suite/src/components/wallet/TransactionItem/InstantStakeBadge.tsx

```diff
@@ -65,7 +65,7 @@ export const InstantStakeBadge = memo(({ transaction, symbol }: InstantStakeBadg
         <Badge size="tiny">
             <Wrapper>
                 <Icon name="lightning" size={spacings.sm} />
-                <FormattedCryptoAmount value={amount} symbol={symbol} />
+                <FormattedCryptoAmount value={amount} symbolExtended={symbol} />
```

Comment:
i would leave it called `symbol`

Reply:
Ok

Reply:
also replaced `symbolExtended` in other components. It's cleaner to use only `symbol`. Anyway `symbolExtended` says nothing..

Reply:
updated in [77f55f2](https://github.com/trezor/trezor-suite/pull/15765/commits/77f55f2c4de1e67c4f0af9773e3911620371a2a6)